stories/shared-components/image-button/ImageButton.stories.tsx

```diff
@@ -27,97 +26,47 @@ import { Status } from 'src/shared/types/status';
-              onClick={toggleImage}
-            />
-          </div>
-          <div className={classNames(styles.imageDescription)}>{status}</div>
-        </div>
 
-        <div className={classNames(styles.codeContent)}>
-          <h3>Usage:</h3>
-          {`import ImageButton, {ImageButtonStatus} from 'src/shared/image-button/ImageButton';`}
-          <br />
-          {`import { ReactComponent as ${icon.imageName} } from '${icon.imagePath}';`}
-          <br />
-          {`
-              <ImageButton status={ImageButtonStatus.${status.toUpperCase()}} 
-                description={'enable/disable'} 
-                image={${icon.imageName}}
-              />
-            `}
-        </div>
-      </div>
-      <div className={classNames(styles.containerStyles)}>
-        <div>Available styles:</div>
-        <div className={classNames(styles.imageCard)}>
-          <div className={classNames(styles.imageHolder)}>
-            <ImageButton
-              status={Status.UNSELECTED}
-              description={'enable/disable'}
-              image={icon.image}
-            />
-          </div>
-          <div className={classNames(styles.imageDescription)}>
-            {Status.UNSELECTED}
-          </div>
-        </div>
-
-        <div className={classNames(styles.imageCard)}>
-          <div className={classNames(styles.imageHolder)}>
-            <ImageButton
-              status={Status.SELECTED}
-              description={'enable/disable'}
-              image={icon.image}
-            />
-          </div>
-          <div className={classNames(styles.imageDescription)}>
-            {Status.SELECTED}
-          </div>
-        </div>
-
-        <div className={classNames(styles.imageCard)}>
-          <div className={classNames(styles.imageHolder)}>
-            <ImageButton
-              status={Status.DISABLED}
-              description={'enable/disable'}
-              image={icon.image}
-            />
-          </div>
-          <div className={classNames(styles.imageDescription)}>
-            {Status.DISABLED}
-          </div>
-        </div>
-      </div>
-    </>
+  return (
+    <div className={styles.defaultStoryContainer}>
+      <ImageButton
+        status={status as ImageButtonStatus}
+        description={`This is image button in ${status} state`}
+        image={DownloadIcon}
+        onClick={toggleImage}
+      />
+      <p>
+        I am an image button, and my status is {status}. Click me to move to the
+        next status
+      </p>
+      <ImageButton
+        status={Status.DISABLED}
+        description="I am a disabled button; but I will still show a tooltip when moused over"
+        image={DownloadIcon}
+        onClick={toggleImage}
+      />
+      <p>
+        I am a disabled button; but I will still show a tooltip when moused over
+      </p>
+      <ImageButton
+        status={Status.DEFAULT}
+        description="I am a disabled button; but I will still show a tooltip when moused over"
```

Comment:
description is wrong here, its not a disabled button.

Reply:
✅